### PR TITLE
[Docs] Update site for 3.8.1

### PR DESCRIPTION
**Pull request by jlprat**
Updating the site for 3.8.1.
As https://home.apache.org is now gone, we can't have a real preview of the site for patch versions as all changes would be visible directly.

### CODE REVIEW

File: 38/js/templateData.js [hunk 1]
```diff
@@ -19,6 +19,6 @@ limitations under the License.
 var context={
     "version": "38",
     "dotVersion": "3.8",
-    "fullDotVersion": "3.8.0",
+    "fullDotVersion": "3.8.1-SNAPSHOT",
```
Comment (showuon) [inline 38/js/templateData.js]: Is this correct? Should be `3.8.1` only?

Comment (jlprat) [reply 38/js/templateData.js]: I think it should be 3.8.1 yes. I'll fix it shortly

Comment (jlprat) [reply 38/js/templateData.js]: fixed

**Review by showuon — APPROVED** [review]
LGTM!

**Comment by mimaison** [comment]
Should we move 3.8.0 out of the supported releases?

**Comment by jlprat** [comment]
I guess so. As 3.8.1 superseeded 3.8.0. I'll push a PR changing it

**Comment by jlprat** [comment]
Here is the PR: https://github.com/apache/kafka-site/pull/638

**Pull request merged**